nicolette-LewNic1/jquery-geoquakes-lab/scripts/app.js

```diff
@@ -0,0 +1,110 @@
+/*Part 1. Rendering Data
+-Data structure: HTTP/REST
+-Lists 1 earthquake 
+
+-having difficulty with map
+*/
+
+var quakes = "https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/4.5_week.geojson";
+
+$(document).ready(function() {
+  console.log("Let's get coding!");
+  let map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 37.78, lng: -122.44},
+	zoom: 3
+  };
+
+  $.ajax({
+	method: 'GET',
+	url: quakes,
+	success: function(response) {
+		console.log(success);
+		var title = response.metadata.title;
+		console.log(response.metadata.title);
+
+		var coordinates = response.features[i].geometry.coordinates
+			console.log(coordinates);
+		var time = response.features.properties.time
+			console.log(time);
+	}
+	error: function() {
+		alert("There was an error getting data");
+	},
+
+	map: map,
+          title: data.features[i].properties.title,
+          icon: {
+            url: "images/earthquake.png",
+            scaledSize: {height: 40, width: 40}
+
+
+
+  });
+
+});
+ 
+
+
+
+
+
+
+
+
+
+
```

Comment:
A LOT of commented code. Make sure to remove excess commented code but overall good job.